bahmnicore-api/src/test/java/org/bahmni/module/bahmnicore/dao/impl/ObsDaoIT.java

```diff
@@ -32,8 +32,7 @@ public void shouldRetrievePatientObs() throws Exception {
 
     @Test
     public void retrieveAllObservationsWhenNoVisitIdsAreSpecified() throws Exception {
-        List<Obs> allObs = obsDao.getObsByPatientAndVisit("86526ed5-3c11-11de-a0ba-001e378eb67a", Arrays.asList("Blood Pressure"), new ArrayList<Integer>(), -1, ObsDaoImpl.OrderBy.ASC, null, false, null, null, null);
-
+        List<Obs> allObs = obsDao.getObsByPatientAndVisit("86526ed5-3c11-11de-a0ba-001e378eb67a", Arrays.asList("Blood Pressure"), new ArrayList<Integer>(), Integer.MAX_VALUE, ObsDaoImpl.OrderBy.ASC, null, false, null, null, null);
```

Comment:
any specific reason we are changing from -1 to Integer.MAX_VALUE? What was the significance of -1

Reply:
@angshu "java.lang.IllegalArgumentException: max-results cannot be negative" is the error we are getting if we pass the "-1" value. This supposed to be a max value parameter of integer type. So replaced that with the  Integer.MAX_VALUE